### PR TITLE
[2.7][Form][Entity] Added "translation_domain" and "choice_translation_domain"

### PR DESCRIPTION
**Pull request by merorafael**
Added "choice_translation_domain" and "translation_domain" documentation.

### CODE REVIEW

**Comment by OskarStark** [comment]
thank you for your work :+1:

**Comment by xabbuh** [comment]
@merorafael Thank you for opening the pull request. Could you also update the other affected form types?

File: reference/forms/types/entity.rst [hunk 1]
```diff
@@ -130,6 +132,26 @@ cast into a string and so must have a ``__toString()`` method.
...
+
+**type**: ``boolean``
+
+This option determines whether the translation of the data obtained will be informed domain ``translation_domain``.
+
+translation_domain
+~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.7
+    The ``translation_domain`` option was introduced in Symfony 2.7.
```
Comment (aitboudad) [inline reference/forms/types/entity.rst]: I'm not sure if this is true

**Comment by aitboudad** [comment]
@merorafael see #5134 for more details

**Comment by merorafael** [comment]
@aitboudad I fixed the "choice_translation_domain". The "translation_domain" is correct?

**Comment by elfen** [comment]
Up.

**Comment by elfen** [comment]
Up.

**Comment by merorafael** [comment]
@xabbuh, @albabar, @OskarStark... the doc is complete?

**Comment by WouterJ** [comment]
Hi @merorafael! I'm very sorry that we, for some reason, kinda ignored this PR. It seems that someone else documented this option in the mean time. Nobody however documented the behaviour of `translation_domain` in these cases. As I like that description, I've taken your changes, edited them a bit and created a new PR (keeping all credits to you): https://github.com/symfony/symfony-docs/pull/6233

Thanks for your contribution!

**Pull request closed without merging**